src/commands/admin/addcourse.ts

```diff
@@ -83,22 +110,13 @@ export default class extends Command {
 			},
 			assignments: ['hw1', 'hw2', 'hw3', 'hw4', 'hw5', 'lab1', 'lab2', 'lab3', 'lab4', 'lab5']
```

Comment:
Would we need these to be initialized upon course creation?

Reply:
imo it doesn't hurt to start it for them

Reply:
Fair fair